<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>1000</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:16px;</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Skema</string>
     </attribute>
     <widget class="QListWidget" name="ScheduleList">
      <property name="geometry">
       <rect>
        <x>432</x>
        <y>250</y>
        <width>551</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>411</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>220</y>
        <width>341</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>491</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Find kursusID</string>
      </property>
      <widget class="QPushButton" name="Uni_knap_2">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>30</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Søg</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Universitet_2">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>35</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Universitet nr</string>
       </property>
      </widget>
      <widget class="QLabel" name="Uni_label_2">
       <property name="geometry">
        <rect>
         <x>127</x>
         <y>35</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Vælg Universitet nr</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Uni_knap_3">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>70</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Ryd</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>371</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Lav en ny forelæsning:</string>
      </property>
      <widget class="QLineEdit" name="Line_date_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Date</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Format: xx:xx:xx</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Line_start_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>110</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Start tidspunkt</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Format: xxxx-xx-xx</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Line_location_2">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Lokation</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>90</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Format: xx:xx:xx</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Line_end_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Slut tidspunkt</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Line_courseid_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>kursusid </string>
       </property>
      </widget>
      <widget class="QPushButton" name="make_class_2">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>160</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Udgiv forelæsning</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>140</y>
        <width>491</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Søg skema</string>
      </property>
      <widget class="QLineEdit" name="Universitet_3">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>35</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>KursusID</string>
       </property>
      </widget>
      <widget class="QLabel" name="Uni_label_3">
       <property name="geometry">
        <rect>
         <x>127</x>
         <y>35</y>
         <width>241</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Vælg Universitet nr</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="xml_knap_3">
      <property name="geometry">
       <rect>
        <x>701</x>
        <y>480</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Udskriv kursus-skema til XML</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_kursusid_2">
      <property name="geometry">
       <rect>
        <x>491</x>
        <y>490</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Kursus nr</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Ændringer</string>
     </attribute>
     <widget class="QListWidget" name="ScheduleList2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>951</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="hent_anmodninger">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Hent Anmodninger</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Line_classid">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>370</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>ClassID nr</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>339</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Besvar ændring</string>
      </property>
     </widget>
     <widget class="QPushButton" name="GodkendAnmodning">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>369</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Godkend anmodning</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AfvisAnmodning">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>429</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Afvis anmodning</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 30pt;</string>
    </property>
    <property name="text">
     <string>UniSkema</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>calendarWidget</tabstop>
  <tabstop>ScheduleList</tabstop>
  <tabstop>hent_anmodninger</tabstop>
  <tabstop>ScheduleList2</tabstop>
  <tabstop>Line_classid</tabstop>
  <tabstop>GodkendAnmodning</tabstop>
  <tabstop>AfvisAnmodning</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
